config/300-clusterchannelprovisioner.yaml

```diff
@@ -26,4 +26,6 @@ spec:
     - all
     - knative
     - eventing
+    shortNames:
+    - ccp
```

Comment:
Add "provisioner" as well?

Reply:
`provisioner` would be better as a category

Reply:
I don't know for sure because I haven't tested it, but it's possible the `provisioner` short name would cause a conflict with a `provisioner` category.